tmk_core/common/eeconfig.h

```diff
@@ -48,6 +48,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #define EECONFIG_RGB_MATRIX_SPEED (uint8_t *)32
 // TODO: Combine these into a single word and single block of EEPROM
 #define EECONFIG_KEYMAP_UPPER_BYTE (uint8_t *)33
+#define EECONFIG_HOTP (uint8_t *)34 // 8 bytes
```

Comment:
For ... sanity's stake, this probably should be set to something like: 
```suggestion
#define EECONFIG_HOTP (uint32_t *)34 // Uses 8 bytes
#define EECONFIG_HOTP2 (uint32_t *)38 // Additional buffer
```

And this way, you could use:
```c
eeprom_update_dword(EECONFIG_HOTP, 0);
eeprom_update_dword(EECONFIG_HOTP2, 0);
```

Reply:
Heh, yeah, that's clearer. I'll switch to that.